projects/Mallard/src/helpers/fetch.ts

```diff
@@ -189,6 +190,30 @@ const fetchWeather = <T>(
     )
 }
 
+export const fetchCacheClear = async (): Promise<boolean> => {
```

Comment:
Can we add a little comment describing what `false` | `true` mean as return values?

Reply:
Done this - but hopefully the tests explain it.